### DISCUSSION TITLE
Can't sign up for GitHub Student Developer Pack.

### DISCUSSION THREAD

**Discussion by SathiraSriSathsara** [community/community#110760] [GitHub Education]
### Hi everyone,

I am a Software Engineering student from Sri Lanka, currently enrolled in Cardiff Metropolitan University through a local institute here in Sri Lanka. I have been trying to sign up for the GitHub Student Developer Pack using my Cardiff Met edu email address. However, it seems that my location is presenting an obstacle in the verification process, preventing me from accessing the plethora of resources available through the pack.

The GitHub Student Developer Pack is an invaluable resource for students like myself, offering access to tools and services that are crucial for our development and learning process. As a part of my course and personal projects, having access to these resources would significantly enhance my ability to learn, develop, and contribute to projects.

I understand that there may be specific verification processes in place, and I am more than willing to provide any additional information or documentation required to prove my student status at Cardiff Metropolitan University.

Is there any way you could assist me in gaining access to the Student Developer Pack, or guide me through a possible solution to this issue? Any assistance or direction you could provide would be greatly appreciated.

**Comment by hyper-ayoub** [comment]
Response #1
hi the pack for students is disabled 
we need to contact the support 
or you can change location when you contact him 

best wishes

**Comment by hyper-ayoub** [comment]
Response #2
https://github.com/orgs/community/discussions/1
hi the pack for students is disabled
we need to contact the support
or you can change location when you contact him

best wishes
